pkg/commands/build/status.go

```diff
@@ -51,21 +65,28 @@ The namespace defaults to the kubernetes current-context namespace.`,
 				return errors.New("no builds found")
 			} else {
 				sort.Slice(buildList.Items, build.Sort(buildList.Items))
-				bld, err := findBuild(buildList, buildNumber, args[0], cs.Namespace)
+				bld, err := findBuild(buildList, buildNumber)
 				if err != nil {
 					return err
 				}
-				return displayBuildStatus(cmd, bld)
+
+				if bom {
+					return displayBOM(cmd, bld, rup, tlsConfig)
+				} else {
+					return displayBuildStatus(cmd, bld)
+				}
 			}
 		},
 	}
 	cmd.Flags().StringVarP(&namespace, "namespace", "n", "", "kubernetes namespace")
 	cmd.Flags().StringVarP(&buildNumber, "build", "b", "", "build number")
+	cmd.Flags().BoolVar(&bom, "bom", false, "only print the built image bill of materials")
```

Comment:
Since the `--bom` flags is not additive in nature to the build status info and is an alternate display, should we consider making `kp build bom` subcommand?

Reply:
the `--bom` flag also mirrors `pack inspect-image --bom` so it could be good to continue trying to be consistent